cmd/pipeline-template/save.go

```diff
@@ -121,6 +123,13 @@ func savePipelineTemplate(cmd *cobra.Command, options *saveOptions) error {
 			saveResp.StatusCode)
 	}
 
+	if len(saveRet) > 0 {
+		taskSucceeded := orca_tasks.TaskSucceeded(saveRet)
```

Comment:
Thoughts on splitting up the PR because this PR is doing 2 things at once? 

`pipelineTemplate status and gate task tracking` feature along with `updating gateapi`

Reply:
talked to @Chinikins, it's fine to combine time together